modin/test/test_utils.py

```diff
@@ -251,7 +251,7 @@ def warns_that_defaulting_to_pandas():
         A WarningsChecker checking for a UserWarning saying that Modin is
         defaulting to Pandas.
     """
-    return pytest.warns(UserWarning, match="defaulting to pandas")
+    return pytest.warns(UserWarning, match="[Dd]efaulting to pandas")
```

Comment:
some of the warnings (e.g. the ones that raised in `df.__reduce__`) starts with a capital "D", so changed the regexp to match both of the cases